sections/02_intro.md

```diff
@@ -1,31 +1,27 @@
+Biology and medicine are rapidly becoming particularly data-intensive with
 respect to both research and practice. A recent comparison of genomics with
 social media, online videos and other data-intensive scientific disciplines
 suggested that the field of genomics alone would equal or surpass other fields
 in data generation and analysis within the next decade
 [@doi:10.1371/journal.pbio.1002195]. These data present new opportunities, but
-also new challenges. We expect that algorithms to automatically extract
-meaningful patterns and provide sufficient context to enable us to act will be
-required.
+also new challenges. We foresee a great need for algorithms that can automatically extract meaningful patterns while providing sufficient context to act, all within data constrained and privacy critical environments.
 
 Concurrent with this explosive growth in biomedical data, a new class of machine
-learning algorithm has become widespread in the domain of image analysis.
+learning algorithm was initially popularized in the domain of image analysis.
 Computer scientists are now building many-layered neural networks from
-collections of millions of images. In a famous example, scientists from Google
+collections of millions of images. In an early famous example, scientists from Google
 demonstrated that a neural network could learn to identify cats simply by
 watching online videos [@doi:10.1109/ICASSP.2013.6639343]. Such approaches,
 termed deep learning, seem like a solution to the challenge presented by the
 growth of data in biomedicine. Perhaps these algorithms could identify the
-biological "cats" hidden in our data - the patterns that exist but that we don't
-know to look for - and could act on them.
+biological "cats" hidden in our data - the patterns unknown to researcher - and could act on them.
 
-Deep learning has transformed image analysis, but researchers' initial forays
-into the use of these techniques in biomedicine have been relatively limited.
+
+Deep learning has since transformed image analysis achieving human-level results in several tasks [@doi:10.1007/s11263-015-0816-y]. In addition, deep learning has been revolutionary in speech recognition and synthesis [arXiv:1412.5567v2,   arXiv:1609.08144], game-playing and reinforcement learning [@doi:10.1038/nature16961, doi:10.1038/nature14236], and natural language processing [   arXiv:1310.4546, arXiv:1609.08144]. In contrast, researchers' initial forays into the use of these techniques in biomedicine have been relatively limited.
 There are certainly numerous promising examples, but we have not yet seen the
-massive convergence on these approaches that occurred in the field of image
-analysis. In this review, we discuss whether this is simply a matter of time or
+massive convergence on these approaches that occurred in other fields. In this review, we examine whether this is simply a matter of time or
```

Comment:
I know `massive` was in there before, but `convergence` is probably sufficient. I suggest removal.